base/anaconda-python-3.8/Dockerfile

```diff
@@ -88,7 +92,11 @@ LABEL name="odh-notebook-base-ubi8-anaconda-python-3.8" \
       summary="Anaconda based Python 3.8 base image for ODH notebooks" \
       description="Anaconda Base Python 3.8 builder image based on UBI8 for ODH notebooks" \
       io.k8s.display-name="Anaconda Python 3.8 base image for ODH notebooks" \
-      io.k8s.description="Anaconda Base Python 3.8 builder image based on UBI8 for ODH notebooks"
+      io.k8s.description="Anaconda Base Python 3.8 builder image based on UBI8 for ODH notebooks" \
+      authoritative-source-url="${AUTHORITATIVE_SOURCE_URL}" \
+      io.openshift.build.commit.ref="${BUILD_COMMIT_REF}" \
+      io.openshift.build.source-location="${AUTHORITATIVE_SOURCE_URL}/tree/${BUILD_COMMIT_REF}/base/anaconda-python-3.8" \
+      io.openshift.build.image="${IMAGE_REGISTRY}:base-c9s-python-3.9" TODO ?????
```

Comment:
I'm not quite sure what value should be put on this place. I wasn't able to find the image in the https://quay.io/repository/opendatahub/workbench-images - did I miss it somehow? :thinking:

Reply:
This particular is not built on the ODH, 
it is exclusively only built downstream.
we should probably clean a few of these.

Reply:
I see. So, do you prefer me to remove just this `io.openshift.build.image` label or all four in this case?

The first three are still valid and could be reused for the downstream part. Although, if there is no image build in upstream, I understand that it doesn't make much sense to have them here either.

Reply:
yeah true,
we can remove all of these:
```
 authoritative-source-url
 io.openshift.build.commit.ref
 io.openshift.build.source-location
 io.openshift.build.image
```
as this more for anyone , who would like to build it, though we wouldn't be building it in upstream.

Reply:
Thank you, Harshad! Updated :slightly_smiling_face: